### PR TITLE
Connect to React

### PR DESCRIPTION
**Pull request by Blackolf931**

### CODE REVIEW

File: TSP/TSP.API/Program.cs [hunk 1]
```diff
@@ -18,3 +18,4 @@ public static void Main(string[] args)
                 });
     }
 }
+
```
Comment (nantonov) [inline TSP/TSP.API/Program.cs:21]: Get rid of this changes

File: TSP/TSP.API/AutoMapper/ApiProfile.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ public ApiProfile()
             CreateMap<Employee, EmployeeViewModel>().ReverseMap();
             CreateMap<EmployeeAddViewModel, Employee>();
             CreateMap<Office, OfficeViewModel>().ReverseMap();
-            CreateMap<OfficeAddViewModel, Office>();
+            CreateMap<OfficeAddViewModel, Office>().ReverseMap();
```
Comment (nantonov) [inline TSP/TSP.API/AutoMapper/ApiProfile.cs:14]: Does ReverseMap() really needed here?

**Review by nantonov — CHANGES_REQUESTED** [review]
Delete react packages from WebApi.

File: TSP/TSP.API/Program.cs (hunk 1 above)
Comment (Blackolf931) [reply TSP/TSP.API/Program.cs]: fix

File: TSP/TSP.API/AutoMapper/ApiProfile.cs (hunk 2 above)
Comment (Blackolf931) [reply TSP/TSP.API/AutoMapper/ApiProfile.cs]: fix

**Pull request closed without merging**